src/managed/OpenLiveWriter.PostEditor/PostEditorPreferencesPanel.cs

```diff
@@ -114,6 +125,16 @@ public PostEditorPreferencesPanel()
 
         }
 
+        private void ButtonBrowserDialog_MouseClick(object sender, MouseEventArgs e)
+        {
+            DialogResult result = folderBrowserDialog.ShowDialog();
```

Comment:
When used, this is only allowing you to choose a folder that's inside your documents folder. There's no way of browsing your full PC.